monai/handlers/mlflow_handler.py

```diff
@@ -68,10 +71,14 @@ class MLFlowHandler:
         epoch_log: whether to log data to MLFlow when epoch completed, default to `True`.
             ``epoch_log`` can be also a function and it will be interpreted as an event filter.
             See ``iteration_log`` argument for more details.
+        dataset_log: whether to log information about the dataset at the beginning. This arg
```

Comment:
I think the arg name should be "log_dataset".

Reply:
Hi @Nic-Ma ,
    I just named `dataset_log` arg this way to keep consistence with the `epoch_log` and `iteration_log` parameters. Would you suggest to change all these parameters to `log_xxx` or just the `log_dataset`?

Thanks,
Bin

Reply:
Deleted this arg.

Reply:
So after removing this `dataset_log`, by default, it will always log the dataset, am I right?